src/widget/form/settings/generalform.cpp

```diff
@@ -84,8 +84,6 @@ static QStringList langs = {"Български",
                             "简体中文"};
 
 static QStringList timeFormats = {"hh:mm AP", "hh:mm", "hh:mm:ss AP", "hh:mm:ss"};
```

Comment:
Right. With that though, there is slight inconsistency, and duplicated time format - i.e. `hh:mm - 15:25` and `HH:mm - 15:25`